src/wallet/rpc/wallet.cpp

```diff
@@ -56,7 +56,7 @@ static RPCHelpMan getwalletinfo()
                         {
                             {RPCResult::Type::NUM, "duration", "elapsed seconds since scan start"},
                             {RPCResult::Type::NUM, "progress", "scanning progress percentage [0.0, 1.0]"},
-                        }, /*skip_type_check=*/true},
```

Comment:
I think we still need this?

Reply:
> I think we still need this?

Wouldn't it be invalid because I took out the `skip_type_check` in `src/rpc/util.h`?

Reply:
Yeah, you'll have to not do that ;)

Reply:
Gotchu, I think that works now?